digest_algorithm.hh

```diff
@@ -28,7 +28,8 @@ namespace query {
 enum class digest_algorithm : uint8_t {
     none = 0,  // digest not required
     MD5 = 1,
-    xxHash = 2,// default algorithm
+    xxHash = 2, // default algorithm
+    legacy_xxHash_without_null_digest = 3,
```

Comment:
Shouldn't you set `legacy_xxHash_without_null_digest` to the old xxHash (2) value, so that old Scylla versions map to legacy_xxHash_without_null_digest during rolling upgrade?

Reply:
Yes, sounds like it.